conans/model/info.py

```diff
@@ -591,8 +591,12 @@ def msvc_compatible(self):
         version = compatible.settings.compiler.version
         runtime = compatible.settings.compiler.runtime
         runtime_type = compatible.settings.compiler.runtime_type
+        cppstd = compatible.settings.compiler.cppstd
```

Comment:
There are packages that ``self.settings.rm_safe("compiler.cppstd")``, so this will crash in those cases

Reply:
Changed to "get_safe" and check for not None